workflows/README.md

```diff
@@ -0,0 +1,310 @@
+# Workflow templates
+
+The content in this folder doesn't _necessarily_ belong in this repository, but we don't have a great place to put it that allows for good discoverability. The GitHub Actions workflows in this folder can be adapted to work with any CI, because the base structure will remain the same.
+
+Example implementation of these workflow files [can be found in `exercism/javascript`][git-javascript].
+
+## **HELP**: this looks like _a lot_ of work :sweat:
+
+The rest of the document is really to explain what's going on in the workflows. If you're in a hurry, and you really want to drop `.travis.yml`, but you can't invest right now in making the PR scripts are optimized, scroll down to the **~10 minute guide** on [**Migrating from Travis**](#migrating-from-travis).
+
+## Track CI actions
+
+The recommended actions are as follows:
+
+1. [`configlet` linting][git-configlet] in order to check `config.json`
+2. check for stubs
+3. check for documentation (`v3` requires new files; this might move to `configlet`)
+4. lint the exercises using a "maintainers" configuration
```

Comment:
Should this be "maintainers'" (ie its the configuration the maintainers have made). I wonder if it would be better renamed to "track-specific configuration"?

Reply:
This should be `maintainers`. Track is very ambigious.

Maintainers _should_ be bound to code *style*, students *should not*.

Reply:
(If you want the wording to be better, PRs appreciated ❤️ 🙉)